orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -46,13 +46,22 @@ RANGERGROUP="$5"
 CLOSECONNECTIONS="$6"
 DATABASENAMES="${@: 7}"
 
+# Root directory for local postgres dump.
+BACKUPS_DIR="/var/tmp/"
+# Directory for the current postgres dump.
+DATE_DIR=${BACKUPS_DIR}/$(date '+%Y-%m-%dT%H:%M:%SZ')
+
 {%- from 'postgresql/settings.sls' import postgresql with context %}
 {% if postgresql.ssl_enabled == True %}
 export PGSSLROOTCERT="{{ postgresql.root_certs_file }}"
 export PGSSLMODE=verify-full
 {%- endif %}
 
 errorExit() {
+  if [ -d "$DATE_DIR" ]; then
+    [[ "$DATE_DIR" == "$BACKUPS_DIR"* ]] && rm -rf -v "$DATE_DIR" > >(tee -a $LOGFILE) 2> >(tee -a $LOGFILE >&2)
```

Comment:
So now that you have `DATE_DIR` defined at the top of the script, meaning that every time the script is run it will always have `DATE_DIR=${BACKUPS_DIR}/...` is there a reason to still check `[[ "$DATE_DIR" == "$BACKUPS_DIR"* ]]`? I personally don't see a case where this would be false anymore and I think it's perfectly safe to only do the `rm` now.

Reply:
@sxxgrc The reason I still add that check is to make sure no one changes the value of `$DATE_DIR` somewhere in the code, for example, if someone changes it to $DATE_DIR = root directory before errorExit*+() is running, then that will remove everything including the root directory. But maybe I overthink and was too cautious with that... I will remove that code!

Reply:
Hi @sxxgrc , just removed that check! Please let me know if it looks good! I will rerun a final manual test after making sure the code looks good to you. TY!